pkg/virt-operator/util/config.go

```diff
@@ -419,6 +426,22 @@ func (c *KubeVirtDeploymentConfig) GetVerbosity() string {
 	return "2"
 }
 
+func (c *KubeVirtDeploymentConfig) GetControllerFlags() map[string]string {
+	return c.ControllerFlags
+}
+
+func (c *KubeVirtDeploymentConfig) GetControllerFlagsAsArray() []string {
+	flags := make([]string, 0)
+
+	for f, v := range c.GetControllerFlags() {
+		str := matchAllCap.ReplaceAllString(f, "${1}-${2}")
```

Comment:
why do we need this  regex and to convert convert to lower case?

Reply:
My original thought here was `userControllerThreads` -> `user-controller-threads`. 

But really the map should look like such
```
VirtControllerFlags: 
  user-controller-threads: "3"
```

I'll remove this.